app/src/main/java/org/wildaid/ofish/ui/home/TimerReceiver.kt

```diff
@@ -0,0 +1,26 @@
+package org.wildaid.ofish.ui.home
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import org.wildaid.ofish.R
+import org.wildaid.ofish.app.CHANNEL_ID
+
+private const val notificationId = 119
+
+class TimerReceiver : BroadcastReceiver() {
```

Comment:
Could it be something like NotificationAlarmSender?

Reply:
Done